src/basic/selinux-util.c

```diff
@@ -32,10 +33,11 @@ DEFINE_TRIVIAL_CLEANUP_FUNC(context_t, context_free);
 #define _cleanup_context_free_ _cleanup_(context_freep)
 
 static int cached_use = -1;
+static int cached_enforced = -1;
 static struct selabel_handle *label_hnd = NULL;
 
-#define log_enforcing(...) log_full(security_getenforce() == 1 ? LOG_ERR : LOG_WARNING, __VA_ARGS__)
-#define log_enforcing_errno(r, ...) log_full_errno(security_getenforce() == 1 ? LOG_ERR : LOG_WARNING, r, __VA_ARGS__)
+#define log_enforcing(...) log_full(mac_selinux_enforced() ? LOG_ERR : LOG_WARNING, __VA_ARGS__)
```

Comment:
Can we please call this `mac_selinux_enforcing`? We already have `log_enforcing`, and `_getenforce` and `enforcing=` on the kernel command line, and yet another spelling makes it harder to remember how to spell the thing.